browser/extensions/BUILD.gn

```diff
@@ -99,9 +101,7 @@ source_set("extensions") {
     "//url",
   ]
 
-  if (toolkit_views) {
-    deps += [ "//brave/components/sidebar/browser" ]
-  }
+  deps += [ "//brave/components/sidebar/browser" ]
```

Comment:
Please move this into deps = []  above. We don't need to add this with +=.

Reply:
Done!